global.json

```diff
@@ -0,0 +1,5 @@
+{
+  "sdk": {
```

Comment:
delete this file since it's in your other PR

Reply:
Ok, done.